inst/schema/scenarioResults.json

```diff
@@ -35,10 +35,6 @@
                 "type": "object",
                 "properties": {
                     "id": { "type":  "string" },
-                    "level": {
-                        "type": "string",
-                        "enum": ["light", "heavy"]
-                    },
```

Comment:
so we're not going to distinguish these any more?

Reply:
Yep, they're going away for IDM as we aren't including the full feature set of switching between levels during the simulation.